FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/V2AppBarActivity.kt

```diff
@@ -0,0 +1,328 @@
+package com.microsoft.fluentuidemo.demos
+
+import android.os.Bundle
+import android.widget.Toast
+import androidx.compose.animation.core.LinearEasing
+import androidx.compose.animation.core.animateFloatAsState
+import androidx.compose.animation.core.tween
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.gestures.detectDragGestures
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.Icon
+import androidx.compose.material.IconButton
```

Comment:
nit: unused import. 
we can use ctrl+alt+o in Android Studio for optimizing imports. It will take care of removing all the unused ones too.

Reply:
Done